### PR TITLE
Add block in node

### PR DESCRIPTION
**Pull request by alavieille**
https://github.com/open-orchestra/open-orchestra-model-interface/pull/281
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/724
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2075

### CODE REVIEW

File: BackofficeBundle/Controller/BlockController.php [hunk 1]
```diff
@@ -25,7 +29,7 @@ class BlockController extends AbstractAdminController
      *
      * @return Response
      */
-    public function newAction(Request $request, $component, $language)
+    public function newSharedAction(Request $request, $component, $language)
```
Comment (ngilain) [inline BackofficeBundle/Controller/BlockController.php]: @alavieille : newSharedBlockAction ?

File: BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Block/BlockView.js [hunk 2]
```diff
@@ -46,6 +52,31 @@ class BlockView extends OrchestraView
...
+     *
+     * @param {Object} event
+     *
+     * @returns {boolean}
+     * @private
+     */
+    _editBlock(event) {
+        event.stopPropagation();
+        if (true === this._block.get('transverse')) {
+            throw new ApplicationError('Blocks transverse is not editable in this context');
```
Comment (ngilain) [inline BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Block/BlockView.js]: @alavieille : blocks are or block is

**Comment by alavieille** [comment]
@ngilain pr updated

**Comment by ngilain** [comment]
:+1:

**Pull request merged**